variables.tf

```diff
@@ -63,20 +63,26 @@ variable "attributes" {
   default = []
 }
 
+variable "global_secondary_indices" {
+  description = "A map defining global secondary indices"
+  type        = any
```

Comment:
Klarer vi ikke å angi typen her?

Reply:
Støtte for optional attributes object types er eksperimentell. https://www.terraform.io/language/expressions/type-constraints#experimental-optional-object-type-attributes

Jeg tror planen er at det kommer i Terraform 1.3. Kan fikse det da.